src/components/admin/filterNumberRange.js

```diff
@@ -0,0 +1,24 @@
+adminApp.filterNumberRange = {
+  view: function(ctrl, args){
+    return m('.w-col.w-col-3.w-col-small-6', [
+            m('label.fontsize-smaller[for="field-6"]', args.label),
+            m('.w-row', [
+              m('.w-col.w-col-5.w-col-small-5.w-col-tiny-5', [
+                m('input.w-input.text-field.positive[name="field-5"][type="text"]', {
```

Comment:
Maybe we should just remove all name attributes. Since things like ```field-5``` are meaningless and I think we don't use the names at all in this situation.